packages/esm-patient-registration-app/src/widgets/delete-identifier-confirmation-modal.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useCallback } from 'react';
+import styles from './delete-identifier-modal.scss';
+import { useTranslation } from 'react-i18next';
+import { Button } from 'carbon-components-react';
+
+interface DeleteIdentifierConfirmationModalProps {
+  deleteIdentifier: (x: boolean) => void;
+  identifierName: string;
+  identifierValue: string;
+}
+
+const DeleteIdentifierConfirmationModal: React.FC<DeleteIdentifierConfirmationModalProps> = ({
+  deleteIdentifier,
+  identifierName,
+  identifierValue,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.modalContent}>
+      <h1 className={styles.productiveHeading}>{t('deleteIdentifierModalHeading', 'Remove identifier?')}</h1>
+      <h3 className={styles.modalBody}>
+        {identifierName}
+        {t(
+          'deleteIdentifierModalText',
+          ' identifier for this patient already contains the following information: ',
+        )}{' '}
+        {identifierValue}
```

Comment:
If there's no design, I would suggest that the text should go `Existing ${identifierName} for this patient is ${identifierValue}`.

Reply:
I have changed the text to 

${identifierName} has a value of ${identifierValue}
Are you sure you want to remove this identifier?

 On the recommendation of Ciaran. Is this a good replacement?
 Thanks!